src/core/routing/__tests__/routes.spec.ts

```diff
@@ -90,10 +90,7 @@ describe("routes", () => {
 
     expect(await routingHandler(postReq)).toEqual(allMethods(postReq));
 
-    const getReq = req({
-      ...postReq,
-      method: "GET"
-    });
```

Comment:
Another benefit of the object param implementation is this merging code. Although it will be rare I think.